vivisect/__init__.py

```diff
@@ -606,6 +611,9 @@ def analyze(self):
         Call this to ask any available analysis modules
         to do their thing...
         """
+        if self.getMeta('GUID') == None:
```

Comment:
Definitely *dont* want to get into the game of having "special" things stored in the file format outside the event stream.

Sounds like maybe we need a getVivGuid() function that checks if it's currently None and generates one ( and it's subsequent change event ) when called?